gammapy/data/observations.py

```diff
@@ -68,6 +68,29 @@ class ObservationCTA(object):
     meta : `~collections.OrderedDict`
         Dictionary to store metadata
 
+    Examples
+    --------
+    Create an observation from the CTA 1DC:
+
+    >>> from gammapy.data import ObservationCTA, EventList, GTI
```

Comment:
Maintaining the example here is tedious for you (with all the >>>) and also for me (can't run it easily).
Suggest to put a Python script in `examples/observation_cta.py` for now, and then once this is useful for users it turns into a notebook.
If you want discoverability from here, you can for now mention the example script from the docstring.
Note that we sometimes link to high-level docs or notebooks from class docstrings, if the exmaple is more than 5 lines.

Reply:
Done